tests/data/jinja/ssp_template.md.jinja

```diff
@@ -23,12 +23,12 @@ Test SSP
 
 {% for control in catalog_interface._control_dict.values() %}
 {% if control.group_id ==  group.id %}
-{# ### {{ catalog_interface.get_label(control.control) }}: {{ control.control.title}}
+{# ### {{ control_io_writer.get_label(control.control) }}: {{ control.control.title}}
```

Comment:
We will also need to update the Jinja template in `cloud-ssp`, otherwise the release will break it, ill create an issue there.